core/src/main/java/org/apache/carbondata/core/scan/collector/impl/DictionaryBasedResultCollector.java

```diff
@@ -241,7 +241,8 @@ private void fillComplexColumnDataBufferForThisRow() {
 
   void fillDimensionData(BlockletScannedResult scannedResult, int[] surrogateResult,
```

Comment:
please add comment for this method, since it becomes more parameters

Reply:
Done